_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -93494,8 +93491,8 @@
 "dBX" = (
 /obj/structure/rack,
 /obj/item/storage/toolbox/mechanical,
-/obj/item/clothing/head/welding,
```

Comment:
my god, a welding helmet, an item that can _also_ be printed from an autolathe (or even just be bought from a youtool), and which logically could have been used in the construction/reconstruction of the station!

Reply:
if it's so easy to get then buy it from a youtool or get one from cargo. Which they might deny making for more interaction.

Reply:
saying "nah fuck off" to someone just because you can isn't really an "interaction"

Reply:
it's just like how the chemists spending 10 seconds to make mutagen for the botanists or saying "nah fuck off" to them for no reason wasn't really an interaction back when botanists relied on chemists for mutagen.

well, I mean, it did _technically_ result in interactions, they just happened to be violent and over-escalated ones.

Reply:
if one side doesn't really lose anything by letting the other side get what they want, it's really hard to get a good, justified conflict to happen- that's part of why friendly antags (who, say, happened to only roll steal and escape alive objectives) suck so much.

wew, that was a tangent, wasn't it?